src/account.rs

```diff
@@ -43,6 +42,7 @@ pub(crate) fn new_account(path: Option<String>) -> Result<()> {
     account_addresses.push(wallet.address().to_string());
     create_accounts_file(&wallet_path, account_addresses)?;
 
-    println!("Wallet public address: {}", wallet.address());
+    println!("Wallet address: {}", wallet.address());
+    println!("Wallet plain address: {}", wallet.address().hash());
```

Comment:
Is the plain address the hash of the pubkey? I don't think that should be printed out by default. Regular end-users should be using bech32 addresses exclusively. Pubkey hash should be something users need to purposefully seek out as advanced users.

Reply:
I added hash of the pubkey, since it is useful/needed while testing in local as (if i am not mistaken) chain config requires that for funding the wallet with initial coins. But this is indeed something an advanced user would use. Maybe we can introduce a flag to `account` command in a future PR such that executing something like `forc-wallet account <account_index> --hashed` would return this.